galsim/phase_screens.py

```diff
@@ -221,6 +221,11 @@ def work(i, atm):
                             significantly faster than computing PSFs with non-frozen-flow
                             atmospheres.  If ``alpha`` != 1.0, then it is required that a
                             ``time_step`` is also specified.  [default: 1.0]
+        seeing_exp:         Power law index for wavelength dependent seeing when using geometric
```

Comment:
We had talked about deriving this value from L0.  Is that possible?  If so, it would make it easier for users who may be using a VonKarman profile and want to get this to match up correctly.

Reply:
Yeah, my memory for how these are related was faulty.  There's a whole slide deck [here](https://docs.google.com/presentation/d/1dzB7xHbTjP1Q4Crbw64JSBjJHxzfnBSHlbU9miCT6WY/edit#slide=id.g17ffb075ae_0_59) showing some attempts to infer a relation, but I'm not confident this can or should be used directly.  Safer to just let the user specify a value I think.